app/components/forms/admin/content/pages-form.js

```diff
@@ -34,7 +34,12 @@ export default Component.extend(FormMixin, {
           rules      : [
             {
               type   : 'empty',
-              prompt : this.l10n.t('Please enter an url')
+              prompt : this.l10n.t('Please enter the path')
+            },
+            {
+              type   : 'regExp',
+              value  : '[^/](.*)',
```

Comment:
You need to pass in a `RegExp` object.

`/[^/](.*)/`

Reply:
@niranjan94 I tried that one also in the first place. But no luck with that.